cpp/src/ray/runtime/object/local_mode_object_store.cc

```diff
@@ -0,0 +1,93 @@
+
+#include <algorithm>
+#include <chrono>
+#include <list>
+#include <thread>
+
+#include <ray/api/ray_exception.h>
+#include "local_mode_object_store.h"
+
+namespace ray {
+namespace api {
+
+void LocalModeObjectStore::PutRaw(const ObjectID &object_id,
```

Comment:
Why are we writing another object store implementation instead of using `src/ray/core_worker/store_provider/memory_store/`?